escpos/impl/epson.py

```diff
@@ -178,6 +178,17 @@ def justify_right(self):
         self.device.write('\x1B\x61\x02')
 
 
+    def set_code_page(self, code_page):
+        """Set code page for character printing. This can be used in combination
+        with bytearray to convert encoding. For example:
+
+            for char in bytearray(unicode_text, 'cp1251'):
+                printer.device.write(chr(char))
+        """
+        assert code_page >= 0 and code_page <= 255
```

Comment:
Can you please, replace `assert` with an `if` and raise a proper `ValueError`?

Reply:
Yeah sure. I didn't want to use `assert` there but I think I went with it because the remainder of the code used them or something.